src/components/academy/grading/index.tsx

```diff
@@ -275,6 +308,24 @@ class Grading extends React.Component<IGradingProps, State> {
     }
     this.gridApi.exportDataAsCsv({ allColumns: true });
   };
+
+  private preProcessData = () =>
```

Comment:
How will this preprocessing help? Also what does it do?

Try not to use generic names like `preprocessData`! It does not allow an external reader to understand what is going on. (add a comment at the very least)

Reply:
Renamed it to `sortSubmissions`